JavaScript/JavaScript.sublime-syntax

```diff
@@ -454,19 +454,19 @@ contexts:
         - match: ;
           scope: punctuation.terminator.statement.js
 
-        - match: (?=\S)
-          push: expression
+        - match: \b(const|let|var)\b
+          scope: storage.type.js
+
+        - include: expression-list
     - include: block-scope
 
   block-scope:
-    - match: '\}'
```

Comment:
I am curious about this change - Removing these lines (462-464) means the `meta.for` scope (or anything that uses `parens-block-scope`) won't pop off immediately after the `}`, but rather when the next non-whitespace character is encountered.